tests_thien/testcases/dashboard/edit_delete_pages_test.py

```diff
@@ -0,0 +1,35 @@
+from tests_thien.testcases.test_base import TestBase
+from tests_thien.pages.dashboard.login_page import LoginPage
+from tests_thien.pages.dashboard.dashboard_page import DashboardPage
+from tests_thien.utilities import utilities, constant
+import pytest
+import logging
+
+class EditDeletePagesTest(TestBase):
+    da_login = LoginPage()
```

Comment:
name should be login_page instead of da_login

Reply:
updated